metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/dataset/DepublishedRecord.java

```diff
@@ -79,13 +82,14 @@ public void setRecordId(String recordId) {
     this.recordId = recordId;
   }
 
-  public DepublicationState getDepublicationState() {
-    return depublicationState;
+  public DepublicationStatus getDepublicationStatus() {
+    return depublicationStatus;
   }
 
-  public void setDepublicationState(
-          DepublicationState depublicationState) {
-    this.depublicationState = depublicationState;
+  public void setDepublicationStatus(DepublicationStatus depublicationStatus) {
+    this.depublicationStatus =
+        depublicationStatus == null ? DepublicationStatus.PENDING_DEPUBLICATION
+            : depublicationStatus;
```

Comment:
This seems like overkill? This method is called just once, and not with a null value.

Reply:
I guess it goes the same for the other fields too so I changed it.